manage.py

```diff
@@ -63,6 +63,28 @@ def runserver(info, host, port, debugger):
                extra_files=reload_on_files)
 
 
+@cli.command()
+def check_highlevel():
+    try:
+       h = db.data.check_highlevel()
+       h_meta = db.data.check_highlevel_meta()
+       print("The no of submissions without highlevel metadata :",
+             h[0][0]-h_meta[0][0])
+    except db.exceptions.DatabaseException as e:
+        click.echo("Error: %s" % e, err=True)
+        sys.exit(1)
+
+
+@cli.command()
+def clean_highlevel():
```

Comment:
It would be nice to be able to optionally specify 1 or more row numbers here to remove, with an optional argument to be able to remove all at once. I think that more often we'll only want to retry one or two failed items instead of all of them.
Consider again the name of these methods. To me, "clean" sounds like we are removing all highlevel items, not just failed ones. Take care of the documentation as well to describe what this method does.